edweiss-app/app/(app)/(tabs)/_layout.tsx

```diff
@@ -36,6 +36,15 @@ const TabLayout: ApplicationLayout = () => {
 					),
 				}}
 			/>
+			<Tabs.Screen
+				name="notifications"
+				options={{
+					title: 'Notifications',
+					tabBarIcon: ({ color, focused }) => (
```

Comment:
Why do you use "" once and '' an other time ?

Reply:
No particular reason, I've seen that it was done like that for the other ones so I decided to do the same.